README.md

```diff
@@ -28,6 +28,10 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
 
 See the output of `deepspeech -h` for more information on the use of `deepspeech`. (If you experience problems running `deepspeech`, please check [required runtime dependencies](native_client/README.md#required-dependencies)).
 
+**Building for Windows**
+
+Go to [build README](examples/net_framework/README.md) to start building DeepSpeech for Windows from source.
+
```

Comment:
I'm not sure if this should be here, you think is Ok or where should I move it? @lissyx

Reply:
Given the doc, I think it's okay to have it separated, since it comes with a lot of different points

Reply:
Maybe we could mode it down a little bit?

We have the Prerequisites section

## Prerequisites

* [Python 3.6](https://www.python.org/)
* [Git Large File Storage](https://git-lfs.github.com/)
* Mac or Linux environment

Maybe it could have one more item. Something like...

## Prerequisites

* [Python 3.6](https://www.python.org/)
* [Git Large File Storage](https://git-lfs.github.com/)
* Mac or Linux environment
* For Windows go to [build README](examples/net_framework/README.md) to start building DeepSpeech for Windows from source.